scripts/train_agents.py

```diff
@@ -464,20 +79,40 @@ def N_X_SP(args) -> None:
         unseen_teammates_len=unseen_teammates_len,
         )
 
+def SPN_1ADV(args) -> None:
+    attack_rounds = 3
+    unseen_teammates_len = 1
+    adversary_play_config = AdversaryPlayConfig.MAP
+    primary_train_types = [TeamType.SELF_PLAY, TeamType.SELF_PLAY_ADVERSARY]
+
+    primary_eval_types = {'generate': [TeamType.SELF_PLAY_HIGH, TeamType.SELF_PLAY_MEDIUM, TeamType.SELF_PLAY_LOW, TeamType.SELF_PLAY_ADVERSARY],
+                          'load': []}
 
-def N_X_SP_w_adversaries(args) -> None:
+    curriculum = Curriculum(train_types = primary_train_types,
+                            is_random = True)
+    get_N_X_SP_agents(
+        args,
+        n_x_sp_train_types=curriculum.train_types,
+        n_x_sp_eval_types=primary_eval_types,
+        curriculum=curriculum,
+        unseen_teammates_len=unseen_teammates_len,
+        adversary_play_config=adversary_play_config,
+        attack_rounds=attack_rounds
+    )
+
+def SPN_1ADV_XSPCKP(args) -> None:
```

Comment:
Also add a comment under here with the description you sent in slack:
" n-x-sp with x teammates. when x is 1, the teammate could be either adv or a sp checkpoint. When x is greater than 2, teammates can be either [adv, spckp, spckp, …] [spckp, spckp, …]"